draft-ietf-quic-tls.md

```diff
@@ -1216,7 +1216,9 @@ negotiation.  TLS uses Application Layer Protocol Negotiation (ALPN)
 {{!RFC7301}} to select an application protocol.  Unless another mechanism is
 used for agreeing on an application protocol, endpoints MUST use ALPN for this
 purpose.  When using ALPN, endpoints MUST abort a connection if an application
```

Comment:
This isn't within your change, but what  does "abort" mean. Should this be "drop QUIC connection state"?

Reply:
I think that an immediate close is what was intended.

Reply:
```suggestion
purpose.  When using ALPN, endpoints MUST immediately close a connection (see Section 10.3 in {{QUIC-TRANSPORT}}) if an application
```